spec/features/item_show_spec.rb

```diff
@@ -32,4 +32,19 @@
     expect(page).to have_selector("#flash_notice")
   end
 
+  context "when a user clicks an item from a previous order thats been retired" do
+    it "notes if item is retired" do
+      user = create(:user)
+      item = Item.create(name: "apple toast", description: "blah", price: 2.00, retired?: true)
+      order = user.orders.create(status: 0, cart: { item.id => 1 })
```

Comment:
Wow, hound caught an actual thing! It looks like line 39 is just creating orders associated with that particular user, and it doesn't need to be stored in a variable.